smoke_tests/tests/authenticated/azure/managed_apps_tests.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from tests.smoke_test_case import SmokeTestCase
+
+
+class ManagedAppsTests(SmokeTestCase):
+    AZURE_SUBSCRIPTION_ID = None
+
+    @staticmethod
+    def managed_apps_url() -> str:
+        return SmokeTestCase.build_bpm_url("/api/azure/v1/managedApps")
+
+    def test_retrieving_managed_apps(self):
```

Comment:
Even if there are no available managed apps, this should fail if the service can't query Azure.
Any functionality such as it returning the correct apps for the user, filtering out unavailable apps unless specified, etc, should all be tested by other means.